### DISCUSSION TITLE
Google vs Treble Experimentations

### DISCUSSION THREAD

**Discussion by claudiojulioferraz** [phhusson/treble_experimentations#1837] [Q&A] [answered]
Hello! Why should I install this rom and not the treble directly from google? Obviously you can’t trust Google, but what’s different about this Rom?

_Automatically translated._

**Comment by haruki-taka8** [comment, marked as answer]
I'm not an expert in GSI ROMs but:

1. I couldn't boot with Google GSIs (Android 11)
2. Phh GSIs provide additional settings on phones (refresh rates, audio policy etc.)
3. Phh GSIs receives more support from the community

and that is why I stick to the ROMs in this repo.

**Reply by claudiojulioferraz** [reply]
Apologies for the delay. Thanks!
